src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -76,6 +77,12 @@ public class Launch4jMojo extends AbstractMojo {
 
     private static final String LAUNCH4J_GROUP_ID = "net.sf.launch4j";
 
+    // intentionally non-static non-final so it can be hacked with reflection if someone really needs to
+    private String DEF_REQADMMAN_RES = "META-INF/resources/manifest-requireAdminRights-v1.xml";
+
+    // intentionally non-static non-final so it can be hacked with reflection if someone really needs to
+    private String DEF_REQADMMAN_FILE = "target/manifest-requireAdminRights.xml";
```

Comment:
Why not to make them parameters with default values? Users can set them via `pom.xml` without a need to hack via reflection.

Reply:
And should this point to `target/classes`? In other case the file won't be included in the final JAR.

Reply:
If you want to, but if a user needs to change it they can just use "manifest" parameter and get exactly same functionality?
I was thinking more about an ability to quickly patch it in a subclass in case there are some compatibility issues with that particular manifest. It works and I used it many times before, but I can't say I'm 100% certain it's fully compliant with whatever Microsoft intended.

Reply:
Doesn't matter, it's for system's use, not the program's. Would /target/classes/META-INF be ok?

Reply:
I think this will be a better place